index.html

```diff
@@ -113,17 +113,70 @@ <h2>Multi-Post Stories</h2>
               </li>
             </ul>
           </div>
-          <div class="project-button">
-            <a href="projectlink"
-              ><img
-                src="images/Icons/Button/Enable.png"
-                alt="button
-        icon"
-            /></a>
+          <div class="project-button click-project featureProj">
+            <button type="button" class="see-project liveLink">
+              See Project
+            </button>
           </div>
         </div>
       </div>
     </section>
+
+    <!-- Pop Up Window -->
+    <div class="popup hidden">
+      <!-- title and the cancel button-->
+      <div class="popup-heading">
+        <h2>Multi Post Stories</h2>
```

Comment:
Please ensure that your Modal popup data is populated with the data from the projects array according to the project requirement.